resources/views/components/profile-loadout.blade.php

```diff
@@ -0,0 +1,67 @@
+<li class="flex flex-row flex-wrap">
```

Comment:
If this is the exact same as our dashboard, could we use the component over there as well?

Reply:
This one is decently customized and has some permissions stuff at the bottom, I don't think they can live in the same one.

Reply:
Can we pull the buttons out of the component and then use the actual detail view portion in both places? That should make it more portable I believe.

Reply:
Hmm...not sure how we would do this.
We call the listing component in the main view: `dashboard-listing`, which has this main utility: 
```php
@foreach($loadoutList as $loadout)
    <x-profile-loadout :loadout="$loadout"/>
@endforeach
```
This has some surrounding HTML which I would argue can be lifted out of this component, and we can scrap that component itself. This alleviates the need to have separate listing components, and allows me to focus on the main area that needs customization: the loadout card itself. Also, there are some stylings in `dashboard-listing` which I needed to manipulate in this specific case (the stylings in there are specific to "dashboard" page). 
Again, these can be lifted out of the component and just call the `dashboard-loadout` in a `foreach` loop at that level

Now, onto the `dashboard-loadout`.  I've styled the actual component itself slightly different to accommodate spacing to give the buttons breathing room.
Then there's the additional `flex-col` I need to have for my buttons themselves.
You could definitely say this belongs in a separate component, but it will be a one-off. Do we really want to pull all this out just to have some one-off components anyways? I would rather know where all the profile logic/styling lives so I can chase it all down at once. If this starts to become logic we need in a lot of other areas, we can work to separate it out -- I just see no value in it for this one scenario.

In summary:
- Recommend removing the `dashboard-listing` component to allow for a separate `profile-loadout` component to be loaded
- Do not recommend separating the different pieces of `profile-loadout` component to support singular use-case